src/soundfile.js

```diff
@@ -328,6 +328,11 @@ define(function (require) {
    * @param {Number} [duration]          (optional) duration of playback in seconds
    */
   p5.SoundFile.prototype.play = function(startTime, rate, amp, _cueStart, duration) {
+    if (!this.output) {
+      console.warn('SoundFile.play() called after dispose');
+      return;
+    }
```

Comment:
I'm wondering why put a warning here, and not other places where a method might be called post-dispose? I'd rather not add complexity to the play method because it's already more complex than it needs to be.

Reply:
this was just the one method that i came across during the tests where user code called a library function after the library objects had been disposed (i believe it was something to do with a loop timer).

the other places I changed were just errors caused during p5 instance teardown, so i figured a warning wasn't really appropriate since they're much more likely to happen and they're not really due to user error.